saleor/checkout/migrations/0070_checkout_customer_note.py

```diff
@@ -0,0 +1,50 @@
+# Generated by Django 3.2.25 on 2024-08-09 08:58
+
+from django.db import migrations, models
+
+CUSTOMER_NOTE_UPDATE_BATCH_SIZE = 100
+
+
+def queryset_in_batches(queryset):
+    """Slice a queryset into batches.
+
+    Input queryset should be sorted by pk.
+    """
+    start_pk = 0
+
+    while True:
+        qs = queryset.filter(pk__gt=start_pk)[:CUSTOMER_NOTE_UPDATE_BATCH_SIZE]
+        pks = list(qs.values_list("pk", flat=True))
+        if not pks:
+            break
+        yield pks
+        start_pk = pks[-1]
+
+
+def copy_note_to_customer_note(apps, schema_editor):
+    Checkout = apps.get_model("checkout", "Checkout")
+
+    queryset = Checkout.objects.filter(note__isnull=False).order_by("pk")
```

Comment:
Shouldn't be a celery task? This could take a while only to filter out all records.